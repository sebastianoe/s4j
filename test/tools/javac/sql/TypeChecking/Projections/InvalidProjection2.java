/*
 * @test
 * @compile/fail/ref=InvalidProjection2.out InvalidProjection2.java -sqlrepo /Users/sebastian/ma/jdk7u-langtools/test/tools/javac/sql/repo.json
 */
// key: compiler.err.sql.validation.types.not.compatible
import java.sql.ResultSet;

public class InvalidProjection2 {
	public static void main(String[] args) {
		String s = SQL[SELECT quantity FROM line_item];
	}
}