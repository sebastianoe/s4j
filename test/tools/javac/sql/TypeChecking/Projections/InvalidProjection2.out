error: The determined sql query type java.lang.Integer cannot be assigned to the expected type java.lang.String
1 error